packages/turf-angle/index.js

```diff
@@ -21,20 +22,18 @@ function angle(startPoint, midPoint, endPoint, options) {
     // Optional Parameters
     options = options || {};
     if (!isObject(options)) throw new Error('options is invalid');
-    var explementary = options.explementary;
-    var mercator = options.mercator;
 
     // Rename to shorter variables
     var A = startPoint;
     var O = midPoint;
     var B = endPoint;
 
-    var azimuthAO = bearingToAzimuth((mercator !== true) ? bearing(A, O) : rhumbBearing(A, O));
-    var azimuthBO = bearingToAzimuth((mercator !== true) ? bearing(B, O) : rhumbBearing(B, O));
+    var azimuthAO = bearingToAzimuth((options.mercator !== true) ? bearing(A, O) : rhumbBearing(A, O));
+    var azimuthBO = bearingToAzimuth((options.mercator !== true) ? bearing(B, O) : rhumbBearing(B, O));
     var angleAO = Math.abs(azimuthAO - azimuthBO);
 
     // Explementary angle
-    if (explementary === true) return 360 - angleAO;
+    if (options.explementary === true) return 360 - angleAO;
```

Comment:
👍 I'm also starting to like using `options.*` instead of defining them at the beginning, it's easier to tell which variables are derived from the options.

Reply:
👍